user-manager-service/pom.xml

```diff
@@ -10,10 +10,13 @@
     </parent>
 
     <groupId>com.crm-food-establishment</groupId>
-    <artifactId>user-manager</artifactId>
+    <artifactId>user-auth-service</artifactId>
```

Comment:
why did you decide to rename this service to 'user-auth'
In the nearest future it will contain functionality used by other services except auth

Reply:
As that service consist of two parts: user service and auth service, so I called it accordingly

Reply:
imo user-manager should be enough
when I see user-auth I think only abt user authentication

Reply:
resolved